remoteDatastore.js

```diff
@@ -0,0 +1,52 @@
+const Buffer = require('safe-buffer').Buffer
+const TreeDAG = require('./datastore.js')
+const fetch = typeof window !== 'undefined' ? window.fetch : require('node-fetch')
```

Comment:
could we add the following to package.json? or does this break how you build?
``` 
"browser": {
    "node-fetch": "./fetch.js"
  }
```

where fetch.js would just be
```
module.exports = window.fetch
```